editor/plugins/script_text_editor.cpp

```diff
@@ -2128,12 +2128,16 @@ void ScriptTextEditor::_color_changed(const Color &p_color) {
 		new_args = String("(" + rtos(p_color.r) + ", " + rtos(p_color.g) + ", " + rtos(p_color.b) + ", " + rtos(p_color.a) + ")");
 	}
 
-	String line = code_editor->get_text_editor()->get_line(color_position.x);
-	String line_with_replaced_args = line.replace(color_args, new_args);
+	String line = code_editor->get_text_editor()->get_line(color_position.y);
+	String line_with_replaced_args = line;
+	int pos = line.find(color_args, color_position.x);
+	if (pos >= 0) {
+		line_with_replaced_args = line.substr(0, pos) + new_args + line.substr(pos + color_args.length(), line.length());
```

Comment:
```suggestion
		line_with_replaced_args = line.substr(0, pos) + new_args + line.substr(pos + color_args.length());
```
Unneeded, and past the end anyway